src/libdrakvuf/win.c

```diff
@@ -244,6 +244,41 @@ static bool find_kernbase(drakvuf_t drakvuf)
     return 1;
 }
 
+addr_t win_get_function_argument(drakvuf_t drakvuf, drakvuf_trap_info_t* info, int narg)
+{
+    page_mode_t pm = drakvuf_get_page_mode(drakvuf);
```

Comment:
Why not just use `drakvuf->pm` directly?

Reply:
I think it is more flexible and powerful way. And in case of LTO it is just for free.
And... I just did not know about this possibility.